source/io/module_eggShell.js

```diff
@@ -351,7 +351,7 @@
             var stack = Module.stackSave();
 
             var type = 'JSON';
-            var valueStackPointer = Module.coreHelpers.writeJSStringToHeap(value);
```

Comment:
I think we should keep the value heap allocated and change the varaible name and add a Module._free

Reply:
because the value strings tend to be large and can blow the stack

Reply:
True, stack could blow if we leave it like that. I'll add a `Module._free`